developerRobot/build.gradle

```diff
@@ -82,7 +82,7 @@ deploy {
                 all {
                     postdeploy << { ctx ->
                         ctx.execute("sync")
-                        ctx.execute("ldconfig")
+                        ctx.execute("sudo ldconfig /home/systemcore/frc/third-party/lib")
```

Comment:
Is this temporary?

Reply:
Yes. Will need to be part of the discussions for how programs launch. But this is one of the easier things to fix once we figure that out.